logger/mask.go

```diff
@@ -159,6 +190,31 @@ func MaskJSON(json, maskChar string, toMasks []MaskData) string {
 	return string(out)
 }
 
+func maskRecursive(obj *fastjson.Value, keys []string, maskChar string, toMask MaskData) {
+	if len(keys) == 0 {
```

Comment:
let's add null validation for `obj` since can be null

Reply:
Done in ebcf419.